### ISSUE TITLE
Deployment to different base urls

### ISSUE THREAD

**Issue by jimmythompson** [srobo/website#6]
Currently the site is hooked up to deploy from github pages, which means the base url has to be `/website`, to match the repository name.

It's unlikely github pages is where we intend to host the site long-term, but it'd be nice to have it still keep working after we move.

I don't know how we can make this work, maybe a different _config.yml for each host? Can we somehow store the common elements only once?

**Comment by jimmythompson** [comment]
Resolved this with 2e129776. 

[_live.yml](https://github.com/srobo/website/blob/gh-pages/_live.yml) puts the site on the root URL. During the build process, we can build the app like so:

```bash
$ jekyll build --config _config.yml,_live.yml
```

The configuration files overwrite each other in order, which is pretty sweet.

**Comment by PeterJCLaw** [comment]
Hi,

Thanks for fixing this. Some practical questions though:
* Is this a standard practise when using Jekyll? (should our readme link to Jekyll's docs?)
* Is this something which can be done via the `rake dev` mentioned in the readme? (is it useful in that scenario?)

I'm somewhat assuming here that this is primarily a dev feature. If so, should it be mentioned somehow in the readme?

Thanks,
Peter

**Comment by jimmythompson** [comment]
It's a pretty standard way of managing multiple environments, which may have different config, in Jekyll.

> "Specify config files instead of using _config.yml automatically. Settings in later files override settings in earlier files." - https://jekyllrb.com/docs/configuration/

I'll note it down on the readme.

Regarding the rake dev, that's a good idea. I'll add it to the task. Right now this is only really for dev, but having environment-specific overrides for Jekyll is always nice to have, especially as we work out how this will sit with the rest of srweb.
